BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -435,7 +435,10 @@ class CbrainTask::<%= name %> < <%= descriptor['cbrain:inherits-from-class'] ||
   def apply_template(template, keys, flags: {}, strip: [])
     keys.inject(template) do |template, (key, value)|
       flag = flags[key].presence
-      next template.gsub(key, flag) if flag && value == true
+  
+      # Flag type
+      isBool = (value.is_a?(TrueClass) || value.is_a?(FalseClass))
+      next template.gsub(key, (flag && value==true)  ? flag : '') if flag && isBool
```

Comment:
I still don't like that we're doing
a) a conditional test
b) a substitution as an argument to `next` and
c) the loop control statement `next` for the inject method

all on a single line... anyway. Just IMHO.